rasa/cli/test.py

```diff
@@ -76,6 +71,12 @@ def add_subparser(
 def _print_core_test_execution_info(args: argparse.Namespace) -> None:
     output = args.out or DEFAULT_RESULTS_PATH
 
+    from rasa.core.test import (
```

Comment:
how about moving these constants to `rasa.core.constants` instead of doing a local import? And if we keep the local import: Can we move it to the top of the function instead of having it in the middle?